Mapbox/MapboxMapsStyleTests/ExpressionTests/ExpressionBuilderTests.swift

```diff
@@ -0,0 +1,37 @@
+import XCTest
+#if canImport(MapboxMaps)
+@testable import MapboxMaps
+#else
+@testable import MapboxMapsStyle
+#endif
+
+internal class ExpressionBuilderTests: XCTestCase {
```

Comment:
Would be good to add a test that checks for a negative key to make sure potential future changes doesn't cause this bug

Reply:
Hmm can't test for that since there's an `assertionFailure` unfortunately. The original bug was also actually caused by an unsorted stops dictionary. That's what's being tested below.